metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py

```diff
@@ -92,7 +92,7 @@
 logger: logging.Logger = logging.getLogger(__name__)
 
 
-def get_platform_from_sqlalchemy_uri(sqlalchemy_uri: str) -> str:
+def get_platform_from_sqlalchemy_uri(sqlalchemy_uri: str) -> str:  # noqa: C901
```

Comment:
Why was this `# noqa: C901` added?

Reply:
The complexity limit, specified for the flake8 check, reached its limit for this function.  The limit is 15 and the additional if is unfortunately number 16. Instead of bumping the limit, I have disabled the check for the function because a rewrite compromised readability, IMO.

Reply:
Makes sense.